core-services/image-mirroring/openshift/mapping_origin_4_15

```diff
@@ -98,6 +98,7 @@ registry.ci.openshift.org/origin/4.15:configmap-reloader quay.io/openshift/origi
 registry.ci.openshift.org/origin/4.15:console quay.io/openshift/origin-console:4.15 quay.io/openshift/origin-console:4.15.0 quay.io/openshift/origin-console:latest
 registry.ci.openshift.org/origin/4.15:console-crontab-plugin quay.io/openshift/origin-console-crontab-plugin:4.15 quay.io/openshift/origin-console-crontab-plugin:4.15.0 quay.io/openshift/origin-console-crontab-plugin:latest
 registry.ci.openshift.org/origin/4.15:console-operator quay.io/openshift/origin-console-operator:4.15 quay.io/openshift/origin-console-operator:4.15.0 quay.io/openshift/origin-console-operator:latest
+registry.ci.openshift.org/origin/4.15:console-plugin quay.io/openshift/origin-console-plugin:4.15 quay.io/openshift/origin-console-plugin:4.15.0 quay.io/openshift/origin-console-plugin:latest
```

Comment:
Here I would suggest to add the `-pipelines-` to the image name since we dont have `openshidft-pipelines` namespace, to avoid any collisions since the `console-plugin` tag is pretty generic.

```suggestion
registry.ci.openshift.org/origin/4.15:console-pipelines-plugin quay.io/openshift/origin-console-pipelines-plugin:4.15 quay.io/openshift/origin-console-pipelines-plugin:4.15.0 quay.io/openshift/origin-console-pipelines-plugin:latest
```